### PR TITLE
GPIB-618

### PR DESCRIPTION
**Pull request by rodonguyen**
- [Link to ticket](https://dltx.atlassian.net/browse/GPIB-618)

image

Personal:
image
image
image

Custodial:
image
image
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -19,5 +19,6 @@
     <div id="root"></div>
     <script type="module" src="https://unpkg.com/ionicons@5.0.0/dist/ionicons/ionicons.esm.js"></script>
     <script nomodule="" src="https://unpkg.com/ionicons@5.0.0/dist/ionicons/ionicons.js"></script>
+    <script src="https://testnet.demo.btcpayserver.org/modal/btcpay.js"></script>
```
Comment (bitcoinbrisbane) [inline public/index.html]: lets remove this guy

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
one small thing

File: public/index.html (hunk 1 above)
Comment (rodonguyen) [reply public/index.html]: deleted in my next commit

**Comment by rodonguyen** [comment]
this should have been implemented in a separate branch and merged into features/bills

File: src/pages/BillsPage.js [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+    
+    await handlePaymentComplete();
+  };
+
+  const pollBillStatus = (id) => {
+    const pollInterval = setInterval(async () => {
+      // const response = await gpib.secure.get(`/bills/${id}`);
+      // console.log(response.data);
+
+      // if (response.data?.paid) {
```
Comment (bitcoinbrisbane) [inline src/pages/BillsPage.js:121]: Whats this guy?

File: src/pages/BillsPage.js [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+      // const amount = Number.parseFloat(response.data.btc).toFixed(8);
+      // setBillBtcAmount(amount);
+      // if (showModal && !payWithGpibCustodialWallet) {
+      //   setPaymentAddress(`${response.data.address}`);
+      //   setBillInstructions(`Please send ${amount} BTC to ${response.data?.address}`);
+      //   pollBillStatus(response.data.id);
+      // }
+
+      ////// Mock pollBillStatus
+      pollBillStatus(9999);
```
Comment (bitcoinbrisbane) [inline src/pages/BillsPage.js:168]: whats all this?

File: src/pages/BillsPage.js [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+    } catch (e) {
+      console.log(e);
+      await handlePaymentComplete(false);
+      actions.setErrors({ hidden: e });
+      actions.setSubmitting(false);
+    }
+  };
+
+  // const uploadPDF = async (e) => {
+  // };
```
Comment (bitcoinbrisbane) [inline src/pages/BillsPage.js:178]: Lets remove

File: src/pages/BillsPage.js [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+          >
+            Add to bill schedule
+          </Button>
+          <Button
+            variant="primary"
+            className="mt-3"
+            // onClick={onUpdatePasswordClick}
+          >
+            Upload a PDF
+          </Button> */}
```
Comment (bitcoinbrisbane) [inline src/pages/BillsPage.js:254]: Lets remove

File: src/pages/BillsPage.js [hunk 6]
```diff
@@ -0,0 +1,295 @@
...
+        </Card>
+        <Card>
+          <div className="d-flex justify-content-between">
+            <h4>Scheduled</h4>
+          </div>
+          {/* <ErrorMessage error={fetchDepositHintsError | errorMessage} /> */}
```
Comment (bitcoinbrisbane) [inline src/pages/BillsPage.js:260]: Lets remove

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
- Create new branch as per the convention
- Address the comments

**Comment by rodonguyen** [comment]
PR moved to [here](https://github.com/dltxio/v2.getpaidinbitcoin.com.au/pull/178)

**Pull request closed without merging**